client/post-editor/editor-ground-control/style.scss

```diff
@@ -59,7 +59,20 @@
 	}
 }
 
+.editor-ground-control__history {
+	display: flex;
+	min-height: 20px;
+	justify-content: space-between;
+	position: relative;
+	padding-left: 16px;
+
+	@include breakpoint( "<660px" ) {
+		display: none;
```

Comment:
I'll do this programmatically rather than lazily in CSS - change coming :)

Reply:
@spen -- should I hold off on review for this one?

Reply:
The change shouldn't be dramatic so feel free to have a first pass - I'll get on this tweak next :)

Reply:
I noticed that `@include breakpoint( "<660px" ) {` is used elsewhere in the file for the surrounding elements ('site' for instance). I'm thinking leaving this as-is would be fine - have you any thoughts or preferences there?

Reply:
IE11's flexbox support is supposed to be kinda buggy, but I tested this in it and it worked fine.

Are the `min-height`, `justify-content`, and `position` properties necessary? I don't see any effects when I disable them, but I haven't tested that broadly. Were they copied over from `editor-ground-control__status`, or leftover from experiments with having a Gridicon in there?

Reply:
> Were they copied over from editor-ground-control__status

Yep! I couldn't see any issue without them but figured I'd leave them to be sure that this element behaved as close as possible to `.editor-ground-control__status`.
Good point about the Gridicon, those rules make sense in that context. Since we don't have one here (yet at least) I'll ditch those rules :) If the element evolves to include icons I could add them back in at that time.

Great catch Ian - thanks!